servers/zts/src/main/java/com/yahoo/athenz/zts/workload/impl/DynamoDBWorkloadRecordStoreFactory.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The Athenz Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.athenz.zts.workload.impl;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.yahoo.athenz.auth.PrivateKeyStore;
+import com.yahoo.athenz.common.server.workload.WorkloadRecordStore;
+import com.yahoo.athenz.common.server.workload.WorkloadRecordStoreFactory;
+import com.yahoo.athenz.zts.ResourceException;
+import com.yahoo.athenz.zts.ZTSConsts;
+import com.yahoo.athenz.zts.cert.impl.DynamoDBClientFetcher;
+import com.yahoo.athenz.zts.cert.impl.DynamoDBClientFetcherFactory;
+import com.yahoo.athenz.zts.notification.ZTSClientNotificationSenderImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DynamoDBWorkloadRecordStoreFactory implements WorkloadRecordStoreFactory {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DynamoDBWorkloadRecordStoreFactory.class);
+
+    @Override
+    public WorkloadRecordStore create(PrivateKeyStore keyStore) {
+
+        final String tableName = System.getProperty(ZTSConsts.ZTS_PROP_WORKLOAD_DYNAMODB_TABLE_NAME);
+        if (tableName == null || tableName.isEmpty()) {
```

Comment:
one general comment here - we should start using the jetty's StringUtil.isEmpty() method here since it handles the case with a single method and is better for code coverage.